jms/1.7/modules/ROOT/pages/index.adoc

```diff
@@ -1,287 +1,70 @@
 = JMS Connector - Mule 4
-:page-aliases: connectors::jms/jms-connector.adoc
+:page-aliases: connectors::jms/jms-connector.adoc, connectors::jms/jms-about.adoc
 
 Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
 
-JMS Connector v1.7
+Anypoint Connector for JMS (Java Message Service) (JMS Connector) enables sending and receiving messages to queues and topics for any message service that implements the JMS specification. JMS is a widely-used API for message oriented middleware. It enables the communication between different components of a distributed application to be loosely coupled, reliable, and asynchronous. The JMS Connector main features include:
```

Comment:
Never use a hyphen after a word ending in -ly, per style ref/English grammar. GLOBAL.

Reply:
```suggestion
Anypoint Connector for JMS (Java Message Service) (JMS Connector) enables sending and receiving messages to queues and topics for any message service that implements the JMS specification. JMS is a widely used API for message-oriented middleware. It enables the communication between different components of a distributed application to be loosely coupled, reliable, and asynchronous. The main features of JMS Connector include:
```